### PR TITLE
fix: credit banner link

### PR DESCRIPTION
**Pull request by leangseu-edx**
fix credit banner link. The changes are explained in the comment.

### CODE REVIEW

File: src/data/services/lms/api.js [hunk 1]
```diff
@@ -78,6 +77,8 @@ export default {
   updateEmailSettings,
   updateEntitlementEnrollment,
   deleteEntitlementEnrollment,
+  logEvent,
   logUpgrade,
   logShare,
+  createCreditRequest,
```
Comment (leangseu-edx) [inline src/data/services/lms/api.js:83]: this wasn't export properly.

File: src/data/services/lms/api.test.js [hunk 2]
```diff
@@ -32,6 +32,11 @@ describe('lms api methods', () => {
   beforeEach(() => {
     jest.clearAllMocks();
   });
+  test('keys identical to module', () => {
+    /* eslint-disable-next-line global-require */
+    const { default: defaultApi, ...rest } = require('./api');
+    expect(Object.keys(rest).sort()).toMatchObject(Object.keys(defaultApi).sort());
```
Comment (leangseu-edx) [inline src/data/services/lms/api.test.js:38]: make sure `import * as api  from 'api'` equal `import api from 'api'`

File: src/containers/CourseCard/components/CourseCardBanners/CreditBanner/views/PendingContent.jsx [hunk 3]
```diff
@@ -6,23 +6,18 @@ import { useIntl } from '@edx/frontend-platform/i18n';
...
-          onClick: createCreditRequest,
-          message: formatMessage(messages.viewDetails),
-        }}
-        message={formatMessage(messages.received, { providerName })}
-        requestData={requestData}
-      />
-    </>
+    <CreditContent
+      action={{
+        href,
```
Comment (leangseu-edx) [inline src/containers/CourseCard/components/CourseCardBanners/CreditBanner/views/PendingContent.jsx:16]: decide `PendingContent` view detail should go to our creditor party.

**Review by nsprenkle — APPROVED** [review]
LGTM

**Pull request merged**